src/components/navigation/IconDropdown.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import * as React from 'react'
+import { StyleSheet, Text } from 'react-native'
+import { type AirshipBridge } from 'react-native-airship'
+
+import { nightText } from '../../styles/common/textStyles.js'
+import { THEME } from '../../theme/variables/airbitz.js'
+import { AirshipDropdown } from '../common/AirshipDropdown.js'
+
+type Props = {
+  bridge: AirshipBridge<void>,
+  backgroundColor: string,
+  imageNode: React.Node,
```

Comment:
General: if image is a vector icon, can use. PNG -> FastImage/uri

Reply:
Same icons are used right now, cannot pass the AntDesignIcon in here. Hard coded the icon into this component instead since same icon is used everywhere right now.